packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts

```diff
@@ -30,14 +35,13 @@ export function accordionItemTemplate(
                         <slot name="heading"></slot>
                     </span>
                 </button>
-                ${startSlotTemplate(options)}
                 ${endSlotTemplate(options)}
-                <span class="icon" part="icon" aria-hidden="true">
+                <span class="expand-collapse-icon" part="expand-collapse-icon" aria-hidden="true">
                     <slot name="expanded-icon">
-                        ${options.expandedIcon || ""}
+                        ${options.expandedIcon || expandedIcon}
```

Comment:
I'm partially curious if we need to require a value for some of these. In some cases, it's common, but I think it's hard to say in all cases that some of these are *required*. In some sense, I wonder how many of these are "*must* have" or required. While in most all cases it's more beneficial for the user experience, I find it hard that any of these be *required* to use the template. If someone wants to use an `:after` element to style part of the template instead of using an SVG, that should be acceptable IMO.

I agree 100% to remove any hardcoded icons, I just don't agree that we should then make them *required* at this level. While Foundation is key to delivering a great experience via the CLI, it's not intended to be tightly coupled to that implementation, it's much broader in scope.

Reply:
Makes sense.